app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java

```diff
@@ -64,6 +65,7 @@ public void whenTagsListIsNotEmpty_ViewHolderIsBinded() {
                 .getValue()
                 .get();
 
+        assertThat(adapter.getTagsList().size(), is(tagList.size()));
```

Comment:
Well, we should be checking if the list size is same as before updating and if the particular list item was updated.

I can see improvement by storing adapter.getTagsList().size() before update and comparing it after update, but i don't think we need a matcher here.